### DISCUSSION TITLE
Can Metalama and PostSharp coexist?

### DISCUSSION THREAD

**Discussion by omiguez** [metalama/Metalama#9] [Q&A] [answered]
Hello,

In our project we have aspects applied for multiple features, and I'd like to know if it is possible to port some of the aspects to Metalama, while keeping others still working with Postsharp, so that the transition can be done gradually.

**Comment by gfraiteur** [comment, marked as answer]
Hello,

In theory PostSharp and Metalama can coexist, but we have not tested this scenario yet.

One possible limitation is that all Metalama aspects will be applied before any PostSharp aspect, therefore you have have problems with aspect ordering if you have aspects from both PostSharp and Metalama to the same declaration.
